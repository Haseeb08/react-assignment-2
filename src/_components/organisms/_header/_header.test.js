import React from "react";
import { create } from "react-test-renderer";
import { render, fireEvent, getByTestId } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Header from "./_header";

jest.mock('../../molecules/_nav-menu/_nav-menu', () => ({
    __esModule: true,
    default: ()=><div>I am mocked</div>
}) );


describe("Header component testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<Header />);
    expect(input.toJSON()).toMatchSnapshot();
  });

});
