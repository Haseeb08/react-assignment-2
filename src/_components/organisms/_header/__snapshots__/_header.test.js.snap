// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component testing Matches the snapshot 1`] = `
<div
  className="makeStyles-header-1"
>
  <div>
    I am mocked
  </div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-text"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      logout
    </span>
  </button>
</div>
`;
